src/net/dmytro/homework7/Main.java

```diff
@@ -0,0 +1,96 @@
+package net.dmytro.homework7;
+
+import java.util.Arrays;
+import java.util.Random;
+import java.util.Scanner;
+import java.util.Objects;
+
+public class Main {
+    public static void main(String[] args){
+       System.out.println(findSymbolOccurance("Hello World", 'l'));
+        System.out.println(findWordPosition("Hello World", "World"));
+        System.out.println(stringReverse("Hello"));
+        System.out.println(isPalindrome("rotator"));
+        wordsGame();
+    }
+    public static int findSymbolOccurance(String line, char symbol ){
+        int counter = 0;
+        for (int i = 0; i < line.length(); i++){
+            if (line.charAt(i) == symbol){
+                counter++;
+            }
+
+        }
+        return counter;
+
+    }
+    public static int findWordPosition(String source, String target){
+        boolean str = source.contains(target);
+        if (str == true){
+            return source.indexOf(target);
```

Comment:
you can leave only this line, it returns -1 by default if string doesnt contain such string.